cms/static/sass/elements/_header.scss

```diff
@@ -118,7 +118,7 @@
       display: block;
 
       img {
-        max-height: ($baseline*2);
+        max-height: ($baseline*2.4);
```

Comment:
@DQuraini we never do changes inline in the sass files in edx-platform, please move this change to the theme repo and I will approve it from there